config.json

```diff
@@ -612,6 +612,18 @@
       "unlocked_by": "rotational-cipher",
       "uuid": "d36ce010-210f-4e9a-9d6c-cb933e0a59af"
     },
+    {
+      "core": false,
+      "difficulty": 5,
+      "slug": "two-bucket",
+      "topics": [
+        "algorithms",
+        "loops",
+        "conditionals"
+      ],
+      "unlocked_by": null,
```

Comment:
It would be great to add an `unlocked_by`, as we will want one for basically all the exercises for nextercism!

The only requirement is that it must be unlocked by a core exercise 🙂

Reply:
Which exercise do you think it should be unlocked by?

Reply:
Hmm, I am not too sure. `triangle` is kind of mathematical with a couple of special rules applied - maybe this?